### DISCUSSION TITLE
LTBEEF inspect multiple at once

### DISCUSSION THREAD

**Discussion by Nintendoboi22** [3kh0/ext-remover#644] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hello, i use LTBEEF inspect to remove gogaurdian and other extensions but i was wondering if i can do multiple extensions at once to make it faster and easier. thanks and thanks for this.

**Comment by jessiejs** [comment]
No, you can't. Read the explanation and think about your question before posting. :)

**Reply by Nintendoboi22** [reply]
Booo

**Comment by alcradite** [comment, marked as answer]
Actually you can if you just put the IDs on separate lines and copy & paste it into the console such as:

chrome.management.setEnabled('extensionid', false)
chrome.management.setEnabled('extensionid', false)
chrome.management.setEnabled('extensionid', false)
chrome.management.setEnabled('extensionid', false)
chrome.management.setEnabled('extensionid', false)
etc
This will disable all the ones you put in the extension id

**Reply by Nintendoboi22** [reply]
Thank you kind sir

**Comment by jessiejs** [comment]
Shit I'm so dumb it says LTBEEF not LTMEAT. But LTBEEF is patched anyways. LTMEAT would not let you do that, as it **does not let you run JS**. Point Blank can run JS tho.

For more info on point blank, see https://github.com/swarm-team

**Reply by Nintendoboi22** [reply]
It’s ltbeef inspect…

**Reply by Aka-but-nice** [reply]
@jessiejs Holy shit, just read the title! There's more than one way to perform LTBEEF :moyai:
